bsip-0058.md

```diff
@@ -67,17 +67,255 @@ related poll worker proposals, including FOR and AGAINST, will be created for vo
+
+Furthermore, the GSP price is dynamically being updated as (a) margin calls are filled, and (b) collateral behind debt positions are updated by borrowers.  [**These updates will permit the dynamic GSP price to move towards the natural feed price.**](#ratchet-effect)  The consequence is that margin calls might be present on the order books and provide additional offers for holders to consider.
+
+|Comparison to Globally Settled Asset|
+|-|
+|Trading of a globally settled asset is also possible.  _Under GSP, the possible presence of margin calls on the DEX order books provides holders with more collateral per smart asset (e.g. more BTS per bitCNY) than under under global settlement. Futhermore, the dynamic GSP adjusts the price of these margin calls which provides holders with collateral offers/prices that are higher than with the force settlement price under global settlement._|
+
+
+### <div id="gsp-activated-force-settling" /> Force-Settling
+
+Besides using the DEX to exchange a smart asset (e.g. bitCNY) to trade for its backing collateral (e.g. BTS), a holder of a smart asset has the ability "force-settle" (FS) a smart asset directly into the collateral subject to a delay of 24 hours and subject to a penalty that is specific to a smart asset (e.g. 5%).  This FS ability is true both when an asset is globally settled or not.  When it is not globally settled, the force settlement price of D units of a smart asset can be force settled for collateral equal to
+
+K = D &times; FSR &div; FP<sub>P</sub><sup>K/D</sup> = D &times; FSR &times; FP<sub>P</sub><sup>D/K</sup>
+
+where
+
+- D: amount of the smart asset that is held
+- FSR: force-settlement ratio (e.g. 95%)
+- FP<sub>P</sub><sup>K/D</sup>: published feed price 24 hours after the force settlement is initiated denominated in terms of smart asset divided by collateral (e.g. 2 bitCNY per BTS)
+- FP<sub>P</sub><sup>D/K</sup>: published feed price 24 hours after the force settlement is initiated denominated in terms of collateral divided by smart asset (e.g. 0.5 BTS per bitCNY)
+
+A GS-protected smart asset can be force-settled just as in normal operations albeit at an amount of collateral that is a function of the GSP price.
+
+K = D &times; FSR &div; (1.01 &times; P<sub>gs</sub><sup>K/D</sup>) = D &times; FSR &times; (P<sub>gs</sub><sup>D/K</sup> &div; 1.01)
+
+_P<sub>gs</sub><sup>K/D</sup> is [**not held constant**](#ratchet-effect) and will vary as the margin called debt positions are [purchased on the open market](#gsp-activated-trading)_.  This can be re-expressed for clarity in terms of the ratio of collateral and debt of the _current_ least collateralized debt position (P<sub>LCDP</sub>).
+
+K = D &times; FSR &div; (1.01 &times; P<sub>LCDP</sub><sup>K/D</sup>) = D &times; FSR &times; (P<sub>LCDP</sub><sup>D/K</sup> &div; 1.01)
+
```

Comment:
Perhaps add a note that as long as there are open margin calls, it is usually better to sell into those instead of force-settling. It happens immediately, it happens at a better price (feed*MSSR), and it may cost lower fees (market fee of collateral asset vs settlement offset of debt asset).

Reply:
I believe there was an offline conversation held, with the result being no change is required at this time. I'll let @pmconrad confirm by resolving this conversation. Thanks